### DISCUSSION TITLE
How to manage programs that auto-update?

### DISCUSSION THREAD

**Discussion by wsw70** [ScoopInstaller/Scoop#5651] [Help and Questions]
Should I disable the auto-update of these programs? Or is there a way to synchronize this with scoop?
There are two essential categories of programs in this case:

- evergreen browsers (Chrome, Firefox)
- vscode

**Comment by rashil2000** [comment]
If possible, disable the auto-updates for them. Scoop tries to do that automatically on a best effort basis, but it's not 100% accurate.
